ui/lib/status.js

```diff
@@ -24,4 +24,14 @@ lstatuses = {
   5: "removed"
 };
 
+// function to determine whether or not application is running or pending
+// status <= 1 or undefined
+isAppRunningQuery = function() {
```

Comment:
I added these functions into status, thinking that they relate to statuses defined at the top of the file. Should they be in some sort of utils?